bitnami/nginx/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx
-version: 5.2.4
+version: 5.3.4
```

The minor bump should start with a new patch
```suggestion
version: 5.3.0
```